rest_api/.jhipster/Subscription.json

```diff
@@ -27,10 +27,15 @@
       "fieldId": 1,
       "fieldName": "subscriptionId",
       "fieldType": "Long"
+    },
+    {
+      "fieldId": 2,
+      "fieldName": "topics",
```

Comment:
Topics would be of String type with hierarchical structure. Should I change 'topics' to 'topic'? In the subscription made by the user, the topics field may refer to more than one topic depending on the wild cards used for the field.